src/SystemServicesContext/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { createContext, FunctionComponent, useContext } from "react"
+
+import useIsBluetoothOn from "./useIsBluetoothOn"
+import useIsLocationOn from "./useIsLocationOn"
+import useIsLocationRequired from "./useIsLocationRequired"
+
+export enum LocationPermissions {
```

Comment:
This is totally cosmetic, but i find union types easier to read:

```suggestion
export type LocationPermissions = "NotRequired" | "RequiredOff" | "RequiredOn"
```

Reply:
Yeah actually I agree, updated.